source/ConversationPanel.cpp

```diff
@@ -404,6 +404,8 @@ void ConversationPanel::Exit()
 	// on the player's missions, or force takeoff from a planet.
 	if(callback)
 		callback(node);
+	if(exitGameOnDone)
+		GetUI()->Quit();
```

Comment:
You can give the conversation panel a callback that quits the UI instead of doing it manually here.

Reply:
I don't understand.  Can you elaborate?

Reply:
There is a function in ConversationPanel called `SetCallback` which you can use to pass a callback. As you can see from the code just above the two lines you added, the callback is called when the panel exits.

So, you can call SetCallback with a lambda that calls Quit. Hope that makes sense

Reply:
I'll look for example usage of `SetCallback` in other files.

Reply:
Fixed